### DISCUSSION TITLE
Custom food creation questions

### DISCUSSION THREAD

**Discussion by gramapp** [jlai/fitness-dashboard#31] [Q&A]
I created a custom food, but having several difficulties... 

Where do I add alternate serving size? (in fit bit dashboard, there was a myriad options for custom foods including alternate serving sizes (ie 1 bar is 100 g), and I think comments and there was classification of what kind of food it is like pastry, bread or something like that)

Where do I enter brand? ( I was able to add the brand in the app but not in the dashboard)

Newly saved custom food is not coming up in the search, tried reloading the page to no avail. Why could this be happening even as this new food is showing up in the app?

![image](https://github.com/user-attachments/assets/8e95f922-882a-40db-9f36-004c2ca3b948)
^^^^^^ cropped line from custom food list in the dashboard

![image](https://github.com/user-attachments/assets/9275e450-dc20-4c3f-b72b-394ebdeedd58)
^^^^^^^ nothing is returned in a search

**Comment by jlai** [comment]
I've added brand to the custom food editor (the API docs seem to be out of date but it seems to accept passing in a brand).

Unfortunately I don't think there's a way to add alternate serving sizes or classification.

Not sure why the custom food isn't coming up for you in search. Seemed to work for me. Anyways, I've added the custom food list to the default options that show up in the list locally (without searching using Fitbit's servers/API) so hopefully that makes it easier to get to.

**Comment by gramapp** [comment]
I did not change anything since I reported the post above, dint even refresh the page and now I got several search hit for the same item.... strange. I did see on some items b4 that search returned same results. But it did not bother me enough yet... to report.

![image](https://github.com/user-attachments/assets/76befc21-2e57-4106-8442-f8ef028224a3)

Thanks for adding the brand now its showing up VVVVVVVV
![image](https://github.com/user-attachments/assets/22c18d07-2938-42be-b338-fd406dd87087)
